src/containers/Ledgers/LedgerMetrics.tsx

```diff
@@ -0,0 +1,120 @@
+import { useTranslation } from 'react-i18next'
+import { Tooltip, useTooltip } from '../shared/components/Tooltip'
+import { renderXRP } from '../shared/utils'
+import PauseIcon from '../shared/images/ic_pause.svg'
+import ResumeIcon from '../shared/images/ic_play.svg'
+import './css/ledgerMetrics.scss'
+import { useIsOnline } from '../shared/SocketContext'
+import { useLanguage } from '../shared/hooks'
+
+const DEFAULTS = {
+  load_fee: '--',
+  txn_sec: '--',
+  txn_ledger: '--',
+  ledger_interval: '--',
+  avg_fee: '--',
+  quorum: '--',
+  nUnl: [],
+}
+
+export const LedgerMetrics = ({
+  data: suppliedData,
+  onPause,
+  paused,
+}: {
+  data: any
+  onPause: any
+  paused: boolean
+}) => {
+  const data = { ...DEFAULTS, ...suppliedData }
```

Comment:
Doesn't this cause a name collision with the input parameter `data` for LedgerMetrics? I'm a bit confused on the naming here.

Reply:
See previous comment